doc/MacInstallation.md

```diff
@@ -29,17 +27,39 @@ And postgresql:
 
 Set a password, then start the service:
 
-    lunchy start postgres
+    brew services start postgresql
 
 Change directory to the metamaps git repository, and run:
 
     bundle install
 
 Copy the .example-env file and rename it to .env. Then modify the DB_USERNAME and DB_PASSWORD values to match the postgres username and password you set
 
+Now use rake to create and set up the database
+
     rake db:create
-    rake db:schema:load
-    rake db:seed
+    rake db:setup
```

Comment:
Yeah i think db:setup does include create.

I originally split this out so people would better understand what the command was doing (in case something went wrong). What do you think of that idea?